pages/ru/snapps/how-to-write-a-snapp.mdx

```diff
@@ -0,0 +1,238 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page({ title: "How to write a snapp" });
+
+<Alert kind="info">
+
+Please note that snapps programmability is not yet available on Mina. These docs
+are a preview of work that is currently in progress.
+
+</Alert>
+
+# How to write a snapp
+
+A Snapp consists of a smart contract and a UI to interact with it. First, we’ll install the Snapp CLI and write a smart contract.
+
+## Write a smart contract
+Your smart contract will be written using <a href="https://github.com/o1-labs/snapp-cli/">Snapp CLI</a>. 
+
+Snapp CLI makes it easy to follow recommended best practices by providing project scaffolding including dependencies such as SnarkyJS, a test framework (Jest), code auto-formatting (Prettier), linting (ES Lint), & more. 
```

Comment:
Can you link up these projects please:
- https://jestjs.io/
- https://prettier.io/
- https://eslint.org/

Reply:
Done